### PR TITLE
HOD Module Updated

### PR DESCRIPTION
**Pull request by pTidke**
HOD Module undertaken use cases working.
UI overhaul to OfficeOfHOD module.
Small changes to Sidebar, Navbar, Scrollbar.

### CODE REVIEW

**Comment by amhndu** [comment]
Please remove Fusion.db from the pull-request and describe here how to apply the migrations, so that the integration team may apply them.

File: FusionIIIT/templates/dashboard/navbar.html [hunk 1]
```diff
@@ -1,138 +1,106 @@
...
 <!-- navbar -->
-
 {% block navBar %}
-    {% include 'dashboard/sidenavbar.html' %}
-
-    <div>
-        <div class="ui stackable massive inverted top attached menu" id="navbar">
-            <a class="item" id="sidebartrigger">
-                <i class="sidebar icon"></i>
+{% include 'dashboard/sidenavbar.html' %}
```
Comment (amhndu) [inline FusionIIIT/templates/dashboard/navbar.html:4]: Why is this file included in the PR? Please remove irrelevant changes.

File: FusionIIIT/templates/dashboard/sidenavbar.html [hunk 2]
```diff
@@ -1,206 +1,167 @@
...
-			<img src="{{request.user.extrainfo.profile_picture.url}}" class="ui small circular centered image">
-		{% else %}
-                	<img src="{% static 'globals/img/zlatan.jpg' %}" class="ui small circular centered image">
-		{% endif %}
-                <br>
-
-                <div class="ui bottom attached inverted" style="text-align: center;">
-                    <!--
-                        TODO: User Details!
+<div class="ui sidebar inverted vertical menu">
```
Comment (amhndu) [inline FusionIIIT/templates/dashboard/sidenavbar.html:3]: Similar irrelevant change.

File: FusionIIIT/templates/globals/base.html [hunk 3]
```diff
@@ -1,143 +1,145 @@
...
-<body id="body">
-
-{% block body %}
-    {% block navBar %}
-    {% endblock %}
-{% endblock %}
-
-    <div id="messages">
-        {% for message in messages %}
+    <head>
```
Comment (amhndu) [inline FusionIIIT/templates/globals/base.html:3]: Another irrelevant change.

File: FusionIIIT/templates/officeModule/officeOfHOD/GossipClient.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import java.io.*;
```
Comment (amhndu) [inline FusionIIIT/templates/officeModule/officeOfHOD/GossipClient.java]: Why is there a java file in here?

File: FusionIIIT/templates/officeModule/officeOfHOD/HelloWorld.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import java.io.*;
```
Comment (amhndu) [inline FusionIIIT/templates/officeModule/officeOfHOD/HelloWorld.java]: Again.

**Review by amhndu — CHANGES_REQUESTED** [review]

File: FusionIIIT/templates/globals/base.html (hunk 3 above)
Comment (pTidke) [reply FusionIIIT/templates/globals/base.html:3]: changed Scrollbar styling here

File: FusionIIIT/templates/dashboard/sidenavbar.html (hunk 2 above)
Comment (pTidke) [reply FusionIIIT/templates/dashboard/sidenavbar.html:3]: Changed inconsistent icon sizes here

File: FusionIIIT/templates/dashboard/navbar.html (hunk 1 above)
Comment (pTidke) [reply FusionIIIT/templates/dashboard/navbar.html:4]: Changed Massive to Large menu.

**Comment by pTidke** [comment]
> Please remove Fusion.db from the pull-request and describe here how to apply the migrations, so that the integration team may apply them.

makemigrations & migrate over "office_module" and "academic_information"
and tell me how to remove Fusion.db from pull request

**Comment by ajwad-shaikh** [comment]
@salarx Should we close this? No relevance now

**Comment by salarx** [comment]
Yeah, close it.
